lib/dead_code_detector/base_method_wrapper.rb

```diff
@@ -52,7 +56,7 @@ def owned_method?(method_name)
 
     # Due to caching, new methods won't show up automatically in this call
     def potentially_unused_methods
-      stored_methods = DeadCodeDetector.config.storage.get(self.class.record_key(klass.name))
+      stored_methods ||= DeadCodeDetector.config.storage.get(self.class.record_key(klass.name))
```

Comment:
Without `stored_methods` being an instance variable, everytime you call it, it will return `nil`, am I right?

Reply:
oh yeah. Very true. I had set this up as an instance variable first and then removed it. Good catch.